scripts/explicit_kwargs_checker_test.py

```diff
@@ -0,0 +1,77 @@
+# coding: utf-8
+#
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For details on how to write such tests, please refer to
+# https://github.com/oppia/oppia/wiki/Writing-Tests
+
+import os
+import sys
+import unittest
+
+_PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+_PYLINT_PATH = os.path.join(_PARENT_DIR, 'oppia_tools', 'pylint-1.7.1')
+sys.path.insert(0, _PYLINT_PATH)
+
+# Since this module needs to be imported after adding Isort path,
+# we need to disable isort for this line.
+# pylint: disable=wrong-import-position
```

Comment:
This does not disable it for the next line; it disables it for all lines after this one. We shouldn't do that. Any disabling should be local and restricted only to the lines that it is meant to affect (and no others).

Reply:
Okay! Got it! I'll enable it for the rest of the statements.  
Thanks!